check_api/src/main/java/com/google/errorprone/ErrorProneOptions.java

```diff
@@ -221,6 +225,10 @@ public boolean isIgnoreSuppressionAnnotations() {
     return ignoreSuppressionAnnotations;
   }
 
+  public boolean ignoreLargeCodeGenerators() {
```

Comment:
While this flag is exposed, it seems the associated filtering logic hasn't been open-sourced. Are there plans to do so? This looks like an interesting alternative to `-XepDisableWarningsInGeneratedCode` (if I understand correctly what it does).